Library/Homebrew/cmd/test-bot.rb

```diff
@@ -590,6 +590,11 @@ def formula(formula_name)
           if bottle_step.passed? && bottle_step.has_output?
             bottle_filename =
               bottle_step.output.gsub(/.*(\.\/\S+#{bottle_native_regex}).*/m, '\1')
+            bottle_rb_filename = bottle_filename.gsub(/\.(\d+\.)?tar\.gz$/, ".rb")
+            bottle_merge_args = ["--merge", "--write", "--no-commit", bottle_rb_filename]
```

Comment:
Ah, I see. Yeh, I guess we could so something smarter there (or just not uninstall it and leave that for `git clean -xdf` to handle)